src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php

```diff
@@ -1067,7 +1067,7 @@ private function addValidationSection(ArrayNodeDefinition $rootNode, callable $e
                             ->validate()->castToArray()->end()
                         ->end()
                         ->scalarNode('translation_domain')->defaultValue('validators')->end()
-                        ->enumNode('email_validation_mode')->values(Email::VALIDATION_MODES + ['loose'])->end()
+                        ->enumNode('email_validation_mode')->values((class_exists(Email::class) ? Email::VALIDATION_MODES : ['html5-allow-no-tld', 'html5', 'strict']) + ['loose'])->end()
```

Comment:
Maybe we can also use the following suggestion, since I believe `email_validation_mode` won't be used if `symfony/mailer` is not installed:  
```suggestion
                        ->enumNode('email_validation_mode')->values(class_exists(Email::class) ? Email::VALIDATION_MODES + ['loose'] : [])->end()
```

Reply:
I would keep it the way you changed it here for the same reason we kept `loose` in #60365, let's better be defensive here as we don't know how people configured their applications

Reply:
can you please create a new issue or a pull request? Thanks